corpus/tchinese/politics.corpus.json

```diff
@@ -0,0 +1,69 @@
+{
+    "politics": [
+        [
+            "你讀過的共產黨員",
+            "是的,馬克思提出了一些有趣的觀察。"
+        ],
+        [
+            "什麼是政府",
+            "理想情況下它是人民的代表。"
+        ],
+        [
+            "什麼是綠色和平組織",
+            "全球性組織推動環保行動。"
+        ],
+        // not good enough translation
+        // [
+        //     "什麼是資本主義",
+        //     "全部或大部分生產和分銷的手段,如土地,工廠,鐵路等,都是私人擁有,並以利潤為目的,原本在充分競爭的條件下,經濟體系。它已被普遍特點是對趨勢財富集中,並在它的後期階段,由大公司的增長,增加政府管制等它也被稱為原則,方法,利益,權力,影響力,投資家等,尤其是那些大增持。"
+        // ],
+        // [
+        //     "什麼是社會主義",
+        //     "從誰想要保持自己的沃爾沃汽車。任何的各種理論或生產和分配由社會或社區,而不是個人的生產資料所有製和運作系統的人,社會的所有成員或社區共享共產主義工作和產品。"
+        // ],
```

Comment:
Unfortunately comments don't exist in json files. These commented out lines will need to be removed before this can be merged.